src/main/java/uk/gov/hmcts/ccd/domain/model/definition/CaseType.java

```diff
@@ -138,4 +141,22 @@ public boolean hasDraftEnabledEvent() {
     public boolean hasEventId(String eventId) {
         return events.stream().anyMatch(event -> event.getId().equals(eventId));
     }
+
+    public List<SearchAliasField> getSearchAliasFields() {
+        return searchAliasFields;
+    }
+
+    public void setSearchAliasFields(List<SearchAliasField> searchAliasFields) {
+        if (searchAliasFields != null) {
+            this.searchAliasFields.addAll(searchAliasFields);
```

Comment:
the method naming is a bit misleading. It's not doing a set it's doing an add. We should rename it to `setSearchAliasFields` to avoid bugs

Reply:
The setter is used by the RestTemplate, just wanted to avoid searchAliasFields to be null, so initialised as ArrayList